examples/docker/census-on-omnisci/census-omnisci.dockerfile

```diff
@@ -46,8 +46,9 @@ RUN conda update -n base -c defaults conda -y && \
     conda config --set channel_priority strict && \
     conda create -n modin --yes --no-default-packages && \
     conda activate modin && \
-    conda install -c intel/label/modin -c conda-forge -c intel \
-        modin "ray>=1.0.0" "numpy==1.20.3" "scikit-learn<0.24.0" "daal4py>=2021.1" dpcpp_cpp_rt && \
+    conda install -c intel/label/modintest -c conda-forge \
+        "omniscidbe4py>=v5.5.0" scikit-learn scikit-learn-intelex && \
+    pip install modin[ray] && \
```

Comment:
why we `pip install` modin instead of using `conda install modin-omnisci -c conda-forge` here?

Reply:
If we decide to use `pyomniscidbe`, then we can use `modin-omnisci` from conda-forge.

Reply:
I think it's already decided.